### DISCUSSION TITLE
Turn off current tab row jumping to bottom line

### DISCUSSION THREAD

**Discussion by DzikiWojt** [doublecmd/doublecmd#1909] [General]
Hey,

I look for the settings to turn off jumping current tab row to the bottom line - which is for me completly unusable and unintuitive.
I tried to find the settings in Options and doublecmd.xml - but I cannot :-(

Can't find something obvious in settings file or there is no such parameter ?

If there is no such parameter - is it possible to add this kind of settings ?

**Comment by Skif-off** [comment]
Hm... I feel stupid, but sorry, I don't understand what you're talking about :) Can you describe the problem in some other way?

P.S. Just in case, [this page](https://doublecmd.github.io/doc/en/help.html) describes most of the DC's interface elements.

**Comment by alexx2000** [comment]
Impossible, it is a system component.

**Reply by Skif-off** [reply]
@alexx2000, can you describe the essence of the problem?

**Reply by alexx2000** [reply]
I guess it is about multiple rows of tabs under Windows.  A row with an active tab always at the bottom in this case.

**Reply by DzikiWojt** [reply]
@alexx2000 describe it well. So I discover multirow tabs but can't use it because of its behavior :-(((
Thanks for the information.
